### ISSUE TITLE
Crash Mod

### ISSUE THREAD

**Issue by d4rk3rr41n** [Delta2Force/MCVmComputers#49]
Hi, i am dark and the minecraft crash when i start the pc and select the iso (windows 98) 

 "The game crashed whilst mouseclicked event handler
Error: org.virtualbox_6_1.VBoxException: VirtualBox error: rc=0x8000ffff The session is not locked (session state: Spawning) (0x8000ffff)"

**Comment by Delta2Force** [comment]
Can you try

**Comment by Delta2Force** [comment]
Did you try inserting the ISO before starting the PC?

**Comment by d4rk3rr41n** [comment]
yes, i already tried but still crashing

**Comment by d4rk3rr41n** [comment]
https://youtu.be/bFacZdPQzEk

**Comment by Delta2Force** [comment]
In the video you insert the ISO after the computer  has started, can you try it again with inserting the iso before starting the pc and send a video of it?

**Comment by d4rk3rr41n** [comment]
https://youtu.be/OlFZP2l0rIA video of crash 2

**Comment by CoccodrillooXDS** [comment]
I've tried to reproduce this bug for 2 times (with 2 differents results) @Delta2Force 
> (**NOTE**: *I rebuilt the mod from the source code yesterday with my modification over the OSType [from Other to Other Windows]*)

[https://youtu.be/ziePOWM4Lt4](https://youtu.be/ziePOWM4Lt4)

> **00:49** - First boot (Normal boot with iso inserted before turning on the PC)
> **01:03** - Crash 1 (Start PC then insert iso immediately)
> **02:17** - Crash 2 (Start PC then let it show the Oracle VirtualBox Logo, then insert iso)

I will also attach the 2 logs:
https://pastebin.com/qQwm9SqM
https://pastebin.com/N9ugnRLm

**Comment by CoccodrillooXDS** [comment]
@d4rk3rr41n a fix to this problem has been merged into the source code. (you may wait for the next update or compile it by yourself 😊)

**Comment by d4rk3rr41n** [comment]
https://youtu.be/yoTnG1ejvJ0 and now not starts Virtual Box :c

**Comment by Delta2Force** [comment]
Fixed in 1.4.7 thanks to @CoccodrillooXDS
